### ISSUE TITLE
Server crashing dedicated server

### ISSUE THREAD

**Issue by Arty-4** [RobertSkalko/Mine-and-Slash#323]
[Crash log.log](https://github.com/RobertSkalko/Mine-and-Slash/files/7629870/Crash.log.log
[debug.log](https://github.com/RobertSkalko/Mine-and-Slash/files/7629872/debug.log)
)

**Comment by RobertSkalko** [comment]
Does it still crash after you delete the configs? 

`[30Nov2021 19:03:12.290] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file ./world/serverconfig/MineAndSlash-Server.toml is not correct. Correcting
[30Nov2021 19:03:12.341] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Preparing level "world"
[30Nov2021 19:03:12.341] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $`

Found this in error

**Comment by Arty-4** [comment]
I don't believe it fixed anything, we're still having issues even after deleting the configs.

[Crash log.log](https://github.com/RobertSkalko/Mine-and-Slash/files/7638337/Crash.log.log)
[debug.log](https://github.com/RobertSkalko/Mine-and-Slash/files/7638338/debug.log)

**Comment by RobertSkalko** [comment]
Mahj is this stil happening?

**Comment by duchuy11891** [comment]
khi nào ra phiên bản 1.16.5 vậy ad
